qiita_pet/handlers/study_handlers/description_handlers.py

```diff
@@ -613,6 +613,35 @@ def delete_raw_data(self, study, user, callback):
 
         callback((msg, msg_level, tab, tab_id, None))
 
+    def delete_prep_template(self, study, user, callback):
+        """Delete the selected prep template
+
+        Parameters
+        ----------
+        study : Study
+            The current study object
+        user : User
+            The current user object
+        callback : function
+            The callback function to call with the results once the processing
+            is done
+        """
+        prep_template_id = int(self.get_argument('prep_template_id'))
+
+        try:
+            PrepTemplate.delete(prep_template_id)
+            msg = ("Prep template %d has been deleted" % prep_template_id)
+            msg_level = "success"
+            tab = 'study_information_tab'
```

Comment:
Probably better to redirect in either case to the raw_data_tab? So from the user point of view the only change on the screen is that the prep template has been remove, rather than move to the RawData tab after the prep template is removed (he probably wants to add another prep template just after removing one).